.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "sdk.coverage.tests"]
 	path = sdk.coverage.tests
 	url = https://github.com/applitools/sdk.coverage.tests.git
-	branch = DotNet_POC
+	branch = DotNet_POC_test
```

Comment:
What's the difference between `DotNet_POC` and `DotNet_POC_test` (and isn't there simply `DotNet`?)

Reply:
It was when updates to sdk.coverage were in DotNet_POC_test branch. They already merged.  Please see these updates in new PR https://github.com/applitools/Eyes.Sdk.DotNet/pull/16  . Please commit it, this PR can be closed.